### PR TITLE
add homework for class 5 and merge latest upstream changes

### PR DESCRIPTION
**Pull request by cs79**
Uploading python script and plots for Class 05 homework (pandas).

### CODE REVIEW

File: alexlee/class05_hw_alexlee.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+# set up column names and read the file FROM URL; changed read_csv to read_table for txt data
+names = ['imdbID', 'title', 'year', 'score', 'votes', 'runtime', 'genres']
+data = pd.read_table('http://bit.ly/cs109_imdb', delimiter='\t', names=names).dropna()
+print "Number of rows: %i" % data.shape[0]
+data.head()  # print the first 5 rows
+
+# skipped pasting in example code of how string.split() method works
+
+clean_runtime = [float(r.split(' ')[0]) for r in data.runtime]
+data['runtime'] = clean_runtime
```
Comment (sinanuozdemir) [inline alexlee/class05_hw_alexlee.py]: You can also combine this into one line! 

data['runtime'] = [float(r.split(' ')[0]) for r in data.runtime]

File: alexlee/class05_hw_alexlee.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+"""
+Interesting fact #2:
+
+Movie runtime is a more widely-dispersed variable for higher scoring movies, almost monotonically so (judging by the scatterplot shape).  This could be reflective of "more opportunity for better content" in longer movies, or the impact of a hidden variable such as budget which, at low levels, might be strongly correlated with short runtime for low-rated movies, but not correlated as much when it increases to higher levels.  It could also just be a spurious correlation that produces an interesting pattern.
+
+"""
+# code for interesting fact #2:
+plt.scatter(data.runtime, data.score, alpha=0.2)
+plt.xlabel('Runtime (minutes)')
+plt.ylabel('IMDB Score')
```
Comment (sinanuozdemir) [inline alexlee/class05_hw_alexlee.py]: It might be interesting to plot the x axis on a scale from 80 to 200, a relationship might become more apparent!

**Comment by sinanuozdemir** [comment]
Be careful in the future to not touch other student's work. It might cause problems and deleted work and no one wants that :)

Your homework looks good. You made a good analysis and you explored the data well!

**Comment by cs79** [comment]
How do I avoid the other-students'-work issue?  I haven't touched anything manually, just pulled Kevin's repo to keep my own fork synced.  Should I just not pull and only push / issue pull requests?

**Comment by justmarkham** [comment]
@cs79 This usually happens if you there is a big delay between when you pull and push. If you pull, then add/commit/push right after that, I *think* you will never have any issues. (It's not always clear to me why certain things happen in Git, so don't sweat it!)

**Comment by cs79** [comment]
Ah ok good to know.  Thanks!

**Pull request merged**